docs/docs/09-analytics.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Analytics
+id: analytics
+keywords: [Analytics, Telemetry]
+---
+
+## Analytics Collection
+
+The Wing CLI collects anonymous usage data in order to help us improve. The intent is to collect data that will help us understand how it is being used throughout the community, with hopes that this will help us to build the right features and improve the right areas.
```

Comment:
```suggestion
The Wing CLI collects anonymous usage data in order to help us improve. The intent is to collect data that will help us understand how it is being used throughout the community, with hopes that this will help us build the right features and improve the right areas.
```